omap-tutorial/README.md

```diff
@@ -192,7 +195,7 @@ InclusiveCriterion  ClosingPlaybackRoutes  :  Media     Modem     Voice     Tone
 InclusiveCriterion  OpenedCaptureRoutes    :  Media     Modem     Voice     MultiChannel
 InclusiveCriterion  OpenedPlaybackRoutes   :  Media     Modem     Voice     Tones        DeepMedia
 InclusiveCriterion  SelectedInputDevice    :  SubMic             Headset    MainMic    Dmic1    Dmic2     Dmic1
-InclusiveCriterion  SelectedOutputDevice   :  Earpiece   Headset        Ihf
+InclusiveCriterion  SelectedOutputDevices   :  Earpiece   Headset        Ihf
 InclusiveCriterion  AudioSource            :  Default          Mic              None              VoiceCall  VoiceCommunication  VoiceDownlink  VoiceUplink
```

Comment:
Should you not point to `AudioCriteria.txt` to get the criterion list ? Having it duplicated means every time it is changed both location needs to be updated.